docs/cloud/cloud-troubleshooting/issue-creating-database.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Creating database issues
+layout: default
+parent: Cloud troubleshooting
+grand_parent: Cloud
+nav_order: 5
+---
+
+# Creating database issues.
+
+## Issue - Your service has been stopped.
+
+You will be unable to create new databases
+
+### Cause
+
+Please contact us at se@featurebase.com for the reason of service being stopped.
+
+### Solution
+
+Please contact us at [se@featurebase.com](mailto:se@featurebase.com) immediately to prevent loss of data and restore your sevice.
+
+## Issue - You have exhausted your trial credit balance.
+
+You will be unable to create new databases
+
+### Cause
+
+You have ran out of your trial credit.
+
+### Solution
+
+Please upgrade to paid user or contact us at [se@featurebase.com](mailto:se@featurebase.com).
+
+## Issue - You have exhausted your credit balance.
```

Comment:
That's not the issue, that's the cause.

The issue is that the user can't create new databases